README.md

```diff
@@ -109,6 +118,16 @@ Interrupt => sysboot ${devtype} ${devnum}:11 any ${scriptaddr} /boot/extlinux.co
 ```
 And that would boot our system on partition 17 (11 in base 16).
 
+Alternatively, use the following set of commands:
```

Comment:
I'd advocate for leaving this out, not sure what it adds.

Reply:
This is actually from the latest release notes: https://github.com/m-weigand/pinenote-debian-recipes/releases/tag/v0.3-beta1

I've followed them and did not try the original command ever 🙂